tests/behat/behat.yml

```diff
@@ -0,0 +1,88 @@
+default:
+  suites:
+    default:
+      contexts:
+        - FeatureContext
+        - LibContext:
+            NoScreenDump: "false"
+        - StepsContext
+        - Drupal\DrupalExtension\Context\DrupalContext
+        - Drupal\DrupalExtension\Context\MinkContext
+        - Drupal\DrupalExtension\Context\MarkupContext
+        - Drupal\DrupalExtension\Context\DrushContext
+  extensions:
+    SensioLabs\Behat\PageObjectExtension: ~
+    emuse\BehatHTMLFormatter\BehatHTMLFormatterExtension:
+     name: html
+     renderer: Behat2
+     file_name: Index
+     print_args: true
+     print_outp: true
+     loop_break: true
+    jarnaiz\JUnitFormatter\JUnitFormatterExtension:
+      filename: report.xml
+      outputDir: "%paths.base%/"
+#    Behat\MinkExtension:
+#      goutte: ~git
+    Drupal\DrupalExtension:
+      blackbox: ~
+      api_driver: 'drush'
+      drush:
+        root: '../../../../../web'
+      text:
+        log_out: 'Log ud'
+        username_field: 'Låner- eller CPR-nummer'
+        password_field: 'Pinkode'
+
+
+chrome:
+  extensions:
+    Behat\MinkExtension:
+      selenium2:
+        wd_host: "http://localhost:4444/wd/hub"
+        browser: chrome
+        capabilities:
+           browser: chrome
+           chrome:
+             {switches:['--start_maximized', "--headless", "--disable-gpu"]}
```

Comment:
It this a normal headless chrome? Would it make sens to add "--no-first-run --disable-translate"!

If yes is this also true for the same line under other configurations in this file.

Reply:
There's been no problems regarding these, but I can put them in and test if it still works.